File: 191_number_of_bits.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        # convert number to binary, then into string
        num = str(bin(n))

        # ignore the 0 and b in the beginning
        num = num[2:]

        totalSetBits = 0
        for bit in num:
            totalSetBits = totalSetBits + int(bit) 
            
        return totalSetBits

File: 191_number_of_bits.c
int hammingWeight(uint32_t n) 
{
    int setBits = 0;  
     
   // Stores binary representation of number.
   int binaryNum[32]; // Assuming 32 bit integer.
   int i=0;
   
   for ( ;n > 0; )
   {
      binaryNum[i++] = n % 2;
      n /= 2;
   }
   
   // Printing array in reverse order.
   for (int j = i-1; j >= 0; j--) 
   {
       if (binaryNum[j] == 1)
       {
           setBits++;
       }

   }

    return setBits;
}